config/manifests/patches/spec-descriptors.yaml

```
@@ -0,0 +1,452 @@
+    - path: networking.hostNetwork
+      description: |
+        Enables host network configuration for pods. Note only one pod is permitted per worker in this configuration.
+      displayName: Host network
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+    - path: networking.nodePort
+      description: |
+        Exposed ports on the Kubernetes node level for forwarding services to Cassandra pods. Most deployments will not use this functionality.
+      displayName: Node Port Network Configuration
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+    - path: networking.nodePort.internode
+      description: |
+        Internode port to use when leveraging Node Port network configuration.
+      displayName: Internode Port
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+    - path: networking.nodePort.internodeSSL
+      description: |
+        Internode SSL port to use when leveraging Node Port network configuration.
+      displayName: Internode SSL Port
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+    - path: networking.nodePort.native
+      description: |
+        Native transport port to use when leveraging Node Port network configuration.
+      displayName: Native Port
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+    - path: networking.nodePort.nativeSSL
+      description: |
+        Native transport SSL port to use when leveraging Node Port network configuration.
+      displayName: Native SSL Port
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+    - path: storageConfig
+      description: |
+        Storage descriptors used during the provisioning and consumption
+        of Persistent Volumes.
+      displayName: Storage Configuration
+    - path: storageConfig.cassandraDataVolumeClaimSpec.storageClassName
+      description: "Name of the storage class where data will reside. This must be \ndefined within the Kubernetes prior to provisioning a datacenter.\n"
+      displayName: Storage Class
+      x-descriptors:
+        - urn:alm:descriptor:io.kubernetes:StorageClass
+    - path: storageConfig.cassandraDataVolumeClaimSpec.accessModes[0]
+      description: |
+        Access mode of the volume
+      displayName: Access Modes
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:select:ReadWriteOnce
+    - path: storageConfig.cassandraDataVolumeClaimSpec.resources.requests.storage
+      description: |
+        Storage requirements for the Cassandra data volume.
+      displayName: Data volume size
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.number
```

I think this has to be a text field given it is represented as a number and the unit.

```suggestion
    - path: storageConfig.cassandraDataVolumeClaimSpec.resources.requests.storage
      description: |
        Storage requirements for the Cassandra data volume. Ex: 2048 Gi
      displayName: Data volume size
      x-descriptors:
        - urn:alm:descriptor:com.tectonic.text
```